ci(ios-e2e): cold-start auth-check bound (5000ms) is intermittent on macos-26 sim

## Symptom

Observed during PR [#328](https://github.com/auerbachb/still-point/pull/328) CI on commit `61ab08c`: `ios-e2e-critical` lane failed with

\`\`\`
StillPointAppUITests.swift:364: error: -[StillPointAppUITests.StillPointAppUITests testVoiceOverLabelsForTimerAndPrimaryButton] : XCTAssertLessThanOrEqual failed: ("12615") is greater than ("5000") - Cold start auth check exceeded documented bound
\`\`\`

The app's own reported `coldStartAuthCheckMs` was 12,615 ms — 2.5× the documented 5,000 ms bound. Same SHA on prior runs (`12d8e98`, `57db5cf`) measured within bound. The PR diff was script + docs only — no app code change — so the variance is sim-side, not a real perf regression.

## Where it lives

The bound is in [`ios/StillPointAppUITests/StillPointAppUITests.swift`](https://github.com/auerbachb/still-point/blob/main/ios/StillPointAppUITests/StillPointAppUITests.swift) — the `coldStartMaxMs` constant (currently `5_000`) is asserted against the root accessibility value `coldStartAuthCheckMs=<n>` in `assertColdStartBound()`. Called by `waitForRoot()` whenever `assertColdStart` defaults to `true`.

## Why this matters now

PR [#324](https://github.com/auerbachb/still-point/issues/324) / [#328](https://github.com/auerbachb/still-point/pull/328) added a smart retry classifier that correctly identifies this failure as a **value assertion** (not a timing-shaped wait) and skips retry. That's the right call for a perf bound — but on macos-26 sims, the bound is intermittently violated by sim-driver overhead rather than a real regression in `AppViewModel.checkAuth()`. The non-retry behavior surfaces these flakes red instead of self-healing.

## Options (tradeoffs)

1. **Widen the bound** to something like 15,000 ms with a comment that 5s was the original aspirational bound on faster sims; macos-26 routinely overshoots. Quick fix; weakens the regression detector.
2. **Make the bound environment-conditional** — looser on macos-26 CI runners, tight on developer machines / pinned sim images. More complex; preserves regression-detection on stable hardware.
3. **Skip the cold-start assertion in tests where it's not the focus** — `assertColdStart: false` is already an option on `waitForRoot()`. Audit which tests really need the bound vs. which inherit it transitively.
4. **Investigate whether 12.6 s is actually the app's cold-start time** vs. sim instrumentation overhead. If the app code itself is taking 12 s, that's a real regression worth fixing instead of widening the bound.

## Acceptance criteria

- [ ] Decide on an option (probably #1 or #3 for now, with #4 as a separate investigation if anyone has cycles)
- [ ] If widening: update `coldStartMaxMs` and add a comment explaining why
- [ ] If conditional: detect macos-26 sim and pick a per-environment bound
- [ ] If skipping: list which tests should keep the bound and which should pass `assertColdStart: false`
- [ ] CI on the change shows the previously-flaking test now passes consistently across at least 3 reruns

## References

- PR [#328](https://github.com/auerbachb/still-point/pull/328) merged `9d8d5ff` — the smart-retry classifier that surfaced this bound as the first non-retriable failure that's actually infra-driven
- Failing CI run [25232287845](https://github.com/auerbachb/still-point/actions/runs/25232287845/job/73987067543)

--- a/docs/testing/e2e-policy.md
+++ b/docs/testing/e2e-policy.md
@@ -190,6 +190,16 @@ Fail thresholds are optional:
 
 Default behavior: report-only, store JSON artifacts.
 
+### iOS cold-start auth-check bound
+
+`StillPointAppUITests` carries an in-test guard (`coldStartMaxMs`) on the
+app-reported cold-start auth-check latency (`coldStartAuthCheckMs`), distinct
+from the `app_boot_seconds` metric above and from XCTest launch overhead.
+
+- **Bound:** 8000ms. Raised from 5000ms in [#334](https://github.com/auerbachb/still-point/issues/334) because the auth-check intermittently exceeded 5000ms on contended macos-26 / iOS-26 simulators, producing infra-shaped flakes rather than real regressions.
+- **Scope:** the assertion runs only on cold-start paths that boot to the auth screen (`seedAuthenticated: false`). Authenticated boots (`seedAuthenticated: true`) and seeded relaunches pass `assertColdStart: false`, since the auth-check there is incidental to what the test asserts (home/settings/session behavior).
+- **Rationale:** keeping the guard on the unauthenticated cold-start path preserves a meaningful latency check where it is the focus, while removing it from authenticated boots eliminates the intermittent failures flagged in #334 (Option 3: skip where cold-start is not the focus).
+
 ## 11) PR merge gating policy
 
 For pull requests, required lanes:

--- a/ios/StillPointAppUITests/README.md
+++ b/ios/StillPointAppUITests/README.md
@@ -22,7 +22,7 @@ This directory contains native iOS UI smoke tests for Issue #193 ("journeys beyo
 - Token expiry re-auth path (`testTokenExpiryRoutesToReauthMessage`)
 - Failed sessions API displays visible error (no silent hang) (`testSessionsFailureShowsVisibleRetryMessage`)
 
-Cold-start acceptance guard used by tests: **auth check must complete within 5,000 ms** (captured via root accessibility value `coldStartAuthCheckMs=<n>`).
+Cold-start acceptance guard used by tests: **auth check must complete within 8,000 ms** (captured via root accessibility value `coldStartAuthCheckMs=<n>`). Raised from 5,000 ms in [#334](https://github.com/auerbachb/still-point/issues/334) to absorb macos-26/iOS-26 simulator contention. The guard is asserted only on `seedAuthenticated: false` cold-start paths (boot to auth); authenticated boots and seeded relaunches pass `assertColdStart: false`.
 
 ## Environment knobs used by tests
 

--- a/ios/StillPointAppUITests/StillPointAppUITests.swift
+++ b/ios/StillPointAppUITests/StillPointAppUITests.swift
@@ -5,7 +5,13 @@ final class StillPointAppUITests: XCTestCase {
     // macos-26/iOS 26 CI. `coldStartMaxMs` is separate: it is the app-reported
     // auth-check latency in `coldStartAuthCheckMs`, not XCTest launch overhead.
     private let launchTimeout: TimeInterval = 45
-    private let coldStartMaxMs = 5_000
+    // Bound bumped 5000 -> 8000ms (issue #334): the app-reported cold-start
+    // auth-check intermittently exceeded 5000ms on contended macos-26 iOS-26
+    // simulators, producing infra-shaped flakes rather than real regressions.
+    // 8000ms keeps a meaningful guard while absorbing simulator contention. The
+    // assertion is also scoped to `seedAuthenticated: false` cold-start paths
+    // only (see `waitForRoot`/`assertColdStart`); authenticated boots skip it.
+    private let coldStartMaxMs = 8_000
 
     override func setUpWithError() throws {
         continueAfterFailure = false
@@ -163,7 +169,7 @@ final class StillPointAppUITests: XCTestCase {
         let app = makeApp(seedAuthenticated: true, resetStore: true)
         app.launch()
 
-        waitForRoot("home", in: app, failureMessage: "Home screen did not appear")
+        waitForRoot("home", in: app, failureMessage: "Home screen did not appear", assertColdStart: false)
         openTab(identifier: "tab.progress", in: app, waitingFor: app.staticTexts["history.title"])
 
         openTab(identifier: "tab.settings", in: app, waitingFor: app.staticTexts["settings.title"])
@@ -175,7 +181,7 @@ final class StillPointAppUITests: XCTestCase {
         let app = makeApp(seedAuthenticated: true, resetStore: true)
         app.launch()
 
-        waitForRoot("home", in: app, failureMessage: "Home screen did not appear")
+        waitForRoot("home", in: app, failureMessage: "Home screen did not appear", assertColdStart: false)
         openTab(identifier: "tab.settings", in: app, waitingFor: app.staticTexts["settings.title"])
 
         let editButton = app.buttons["settings.usernameEditButton"]
@@ -203,7 +209,7 @@ final class StillPointAppUITests: XCTestCase {
         let app = makeApp(seedAuthenticated: true, resetStore: true)
         app.launch()
 
-        waitForRoot("home", in: app, failureMessage: "Home screen did not appear")
+        waitForRoot("home", in: app, failureMessage: "Home screen did not appear", assertColdStart: false)
         openTab(identifier: "tab.settings", in: app, waitingFor: app.staticTexts["settings.title"])
 
         let editButton = app.buttons["settings.usernameEditButton"]
@@ -234,7 +240,7 @@ final class StillPointAppUITests: XCTestCase {
         let app = makeApp(seedAuthenticated: true, resetStore: true, forceUsernameConflict: true)
         app.launch()
 
-        waitForRoot("home", in: app, failureMessage: "Home screen did not appear")
+        waitForRoot("home", in: app, failureMessage: "Home screen did not appear", assertColdStart: false)
         openTab(identifier: "tab.settings", in: app, waitingFor: app.staticTexts["settings.title"])
 
         let editButton = app.buttons["settings.usernameEditButton"]
@@ -322,7 +328,7 @@ final class StillPointAppUITests: XCTestCase {
         let app = makeApp(seedAuthenticated: true, resetStore: true, sessionSeconds: 60, timerMultiplier: 1.0)
         app.launch()
 
-        waitForRoot("home", in: app, failureMessage: "Home screen did not appear")
+        waitForRoot("home", in: app, failureMessage: "Home screen did not appear", assertColdStart: false)
         let beginButton = app.buttons["home.beginButton"]
         XCTAssertTrue(beginButton.waitForExistence(timeout: 5))
         tap(beginButton, thenWaitForRoot: "session", in: app)
@@ -349,7 +355,7 @@ final class StillPointAppUITests: XCTestCase {
         let app = makeApp(seedAuthenticated: true, resetStore: true)
         app.launch()
 
-        waitForRoot("home", in: app, failureMessage: "Home screen did not appear")
+        waitForRoot("home", in: app, failureMessage: "Home screen did not appear", assertColdStart: false)
         let beginButton = app.buttons["home.beginButton"]
         XCTAssertTrue(beginButton.waitForExistence(timeout: 5))
         XCTAssertTrue(beginButton.isHittable, "Home primary action should be visible above safe-area/home indicator")
@@ -361,7 +367,7 @@ final class StillPointAppUITests: XCTestCase {
         let app = makeApp(seedAuthenticated: true, resetStore: true, sessionSeconds: 600, timerMultiplier: 0.05)
         app.launch()
 
-        waitForRoot("home", in: app, failureMessage: "Home screen did not appear")
+        waitForRoot("home", in: app, failureMessage: "Home screen did not appear", assertColdStart: false)
         let beginButton = app.buttons["home.beginButton"]
         XCTAssertTrue(beginButton.waitForExistence(timeout: 5))
         XCTAssertEqual(beginButton.label, "Start session")
@@ -381,7 +387,7 @@ final class StillPointAppUITests: XCTestCase {
         let app = makeApp(seedAuthenticated: true, resetStore: false, forceSessionsFailure: true, forceProgressTab: true)
         app.launch()
 
-        waitForRoot("home", in: app, failureMessage: "Home screen did not appear")
+        waitForRoot("home", in: app, failureMessage: "Home screen did not appear", assertColdStart: false)
         let errorLabel = app.staticTexts["history.errorMessage"]
 
         XCTAssertTrue(errorLabel.waitForExistence(timeout: 8))

